includes/discovery/processors/terra.inc.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+if ($device["os"] === "terra") {
+    $descr = "Processor";
+
+    $query = array(
+       array("sti410C", ".1.3.6.1.4.1.30631.1.9.1.1.3.0"),
+       array("sti440",  ".1.3.6.1.4.1.30631.1.18.1.326.3.0")
+    );
+
+    $sysDescr = strlen($device["sysDescr"]) == 0 ? snmp_get($device, ".1.3.6.1.2.1.1.1.0", "-Ovq") : $device["sysDescr"];
```

Comment:
I've just tested this and adding a device does populate the sysDescr value straight away.